src/main/java/com/minecolonies/coremod/client/gui/modules/WindowListRecipes.java

```diff
@@ -130,6 +132,8 @@ private void removeClicked(final Button button)
     {
         final int row = recipeList.getListElementIndexByPane(button);
         final IRecipeStorage data = module.getRecipes().get(row);
+        module.removeRecipe(row);
+        recipeList.refreshElementPanes();
```

Comment:
Is it intended that this refresh is before the server message but the other two are after?

Reply:
I thought order shouldn't matter particularly, so I didn't double check